lua/lazyvim/plugins/extras/lang/kotlin.lua

```diff
@@ -0,0 +1,96 @@
+return {
+  -- Add linting, debug adapter, language server
+  {
+    "williamboman/mason.nvim",
+    opts = function(_, opts)
+      vim.list_extend(opts.ensure_installed or {}, {
+        "ktlint",
+        "kotlin-language-server",
+        "kotlin-debug-adapter",
+      })
+    end,
+  },
+  -- Add syntax highlighting
+  {
+    "nvim-treesitter/nvim-treesitter",
+    opts = function(_, opts)
+      vim.list_extend(opts.ensure_installed or {}, { "kotlin" })
+    end,
+  },
+  -- Add language server
+  {
+    "neovim/nvim-lspconfig",
+    dependencies = "williamboman/mason.nvim",
+    opts = {
+      servers = {
+        kotlin_language_server = {},
+      },
+    },
+  },
+  -- Add linting
+  {
+    "mfussenegger/nvim-lint",
+    dependencies = "williamboman/mason.nvim",
+    opts = {
+      linters_by_ft = { kotlin = { "ktlint" } },
+    },
+  },
+  -- Add formatting
+  {
+    "stevearc/conform.nvim",
+    dependencies = "williamboman/mason.nvim",
```

Comment:
```suggestion
    "stevearc/conform.nvim",
    optional = true,
```
`dependencies` are already defined for the plugin. `optional = true` in case someone might choose to use the Extra `none-ls`. It would also be good if you could provide a similar configuration for `none-ls` if possible for formatting.

Reply:
Oh, I see.

I'm sorry for the confusion. Could you please clarify how to ensure that dependencies have been added to Mason? 

I want to make sure they are properly integrated and stable.

I've checked [here](https://github.com/LazyVim/LazyVim/blob/97480dc5d2dbb717b45a351e0b04835f138a9094/lua/lazyvim/plugins/formatting.lua), but I'm unsure where this module is being called from.

Of course, I'll add none-ls！！！

Reply:
> optional = true in case someone might choose to use the Extra none-ls. It would also be good if you could provide a similar configuration for none-ls if possible for formatting.

Thank you very much for your suggestion.

Of course, I'll add none-ls！！！

Reply:
You can see in the main LazyVim spec for `conform` that mason is already a dependency. This happens [here](https://github.com/LazyVim/LazyVim/blob/97480dc5d2dbb717b45a351e0b04835f138a9094/lua/lazyvim/plugins/formatting.lua#L35-L36). So, there's no need for you to redefine the dependency, since it's already there.

Reply:
oh thanks!!! 

I'll remove the dependencies based on what we saw in the formatting.lua!